sass/style.scss

```diff
@@ -7,115 +7,6 @@
  *   green: #7DD36B
```

Comment:
You can probably remove this comment variable block since these are now nicely consolidated into variables.